content/about/limits.md

```diff
@@ -17,13 +17,35 @@ Unless otherwise negotiated as a part of an enterprise level contract, all Worke
 
 **Note:** app Workers scripts do not count towards this limit.
 
-## Number of Requests Limit
+## Requests Limits
 
 Unlimited (Paid) Workers scripts automatically scale onto thousands of Cloudflare edge servers around the world; there is no general limit to the number of requests per second Workers can handle.
 
 Cloudflare's abuse protection methods do not affect well-intentioned traffic. However, if you send many thousands of requests per second from a small number of client IP addresses, you can inadvertently trigger Cloudflare's abuse protection. If you expect to receive `1015` errors in response to traffic or expect your application to incur these errors, contact Cloudflare to increase your limit.
 
-Accounts using the Workers free tier are limited to a maximum of 100,000 requests per day, with a burst rate limit of 1000 requests per 10 minutes. This limit applies at the account level, meaning that requests on your workers.dev subdomain count toward the same limit as your zoned domains. Visitors who run into the rate limit will be served a Cloudflare 1015 error page, however if you are calling your script programmatically, you can detect the rate limit page and handle it yourself by looking for HTTP status code 429. Upgrading to a paid plan will automatically lift this limit.
+Accounts using the Workers free tier are subject to a burst rate limit of 1000 requests per 10 minutes and a daily request limit of 100,000 requests.
+
+The burst rate and daily request limits apply at the account level, meaning that requests on your workers.dev subdomain count toward the same limit as your zones. Upgrade to a paid plan to automatically lift these limits.
+
+### Burst Rate Limit
+
+Users visiting a burst rate limited site will receive a Cloudflare 1015 error page. However if you are calling your script programmatically, you can detect the rate limit page and handle it yourself by looking for HTTP status code 429.
+
+### Daily Request Limit
+
+Workers that fail as a result of daily request limit errors can be configured by setting its corresponding Route in two modes: _Fail open_ and _Fail closed_.
```

Comment:
Does this language relating Workers and Routes blend with the rest of the docs?  What would be a good place to link to for more route information?